.github/workflows/promote_xmCloud.yml

```diff
@@ -0,0 +1,50 @@
+name: Promote an XM Cloud deployment to a different environment.
+
+on:
+  workflow_call:
+    inputs:
+      environmentName:
+        required: true
+        type: string
+      deploymentId:
+        required: true
+        type: string
+    secrets:
+      XM_CLOUD_CLIENT_ID:
+        required: true
+      XM_CLOUD_CLIENT_SECRET:
+        required: true
+      XM_CLOUD_ENVIRONMENT_ID:
+        required: true  
+    
+jobs:
+
+  deploy:
+    name: Promoting XM Cloud Deployment ${{ inputs.deploymentId }} to ${{ inputs.environmentName }} Site
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '6.0.x' 
```

Comment:
@robearlam shouldn't we push this to 8?

Reply:
Currently the CLI is only built to .NET6, once it's been updated to v8 we will.

We're currently tracking this on #306